docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+networks:
+  cowardscloud_web:
+    name: cowardscloud_web
+    external: true
+
+services:
+  web:
+    image: registry.baileys.dev/devhouse:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+    networks:
+      - cowardscloud_web
```

Comment:
is this attachable for external services?

Reply:
```
dylan@dylans-node-dev ~ 
> $ docker network inspect cowardscloud_web -f "{{ .Attachable }}"
false
```

Reply:
this isn't an external service. it lives on the cowards cloud

Reply:
it's external to the stack

Reply:
"the stack"? it is it's own stack defined in the docker-compose.yaml. you can mount any external overlay network